src/chart/graph/createView.ts

```diff
@@ -77,7 +77,7 @@ export default function createViewCoordSys(ecModel: GlobalModel, api: ExtensionA
             const viewWidth = viewRect.width;
             const viewHeight = viewRect.height;
 
-            const viewCoordSys = seriesModel.coordinateSystem = new View();
```

Comment:
The parameters of coordinate system constructors should be as consistent as possible. The `api` can be an parameter of `setCenter`. 

Or a better solution I think is convert percent value to absolute value before `setCenter`. In some context like 
https://github.com/apache/echarts/blob/4a52199842039d7fa6019ee8584d1a395fbc771a/src/action/roamHelper.ts#L82
It will always be a calculated absolute value

Reply:
I prefre first one